component styles/Alert.css

```diff
@@ -0,0 +1,49 @@
+.alert-container{
+    width: 90vw;
+    margin: 0 auto;
+    max-width: 40rem; 
+}
+
+.alert-container h1,h3{
+    margin-top: 2rem;
+}
+
+.alert-container p{
+    margin: 1rem 0 1rem 0;
```

Comment:
you can use short-hand property, everything else looks fine

Reply:
Ok thank you

Reply:
Ya, margin: 1rem 0; will work the same